SimCalorimetry/HcalTrigPrimProducers/src/HcalTrigPrimDigiProducer.cc

```diff
@@ -19,7 +19,7 @@
 #include "CondFormats/HcalObjects/interface/HcalElectronicsMap.h"
 #include "CondFormats/HcalObjects/interface/HcalLutMetadata.h"
 #include "Geometry/Records/interface/CaloGeometryRecord.h"
-
+#include "TH2.h"
```

Comment:
@davidlange6 that must be a mistake.  I've asked Michael to remove it and update the pr.